meteor-app/imports/ui/assessment/assessment-job-card-list.js

```diff
@@ -100,9 +100,9 @@ export default withTracker(props => {
 
   const renderJob = () => {
     if (statusLine == '') {
-      return Assessment.find({ search: { $regex: searchLine } }).fetch()
+      return Assessment.find({ search: { $regex: searchLine, $options: "i" } }).fetch()
```

Comment:
I think this is a user entered string, and we are asking it to be treated as a regular expression (with case insensitivity) to find something that matches. 

On the surface that seems sensible, but just try entering `\` into the search box and see what happens.